example/lib/identity.dart

```diff
@@ -0,0 +1,192 @@
+    try {
+      identityVersion = await Identity.extensionVersion;
+    } on PlatformException {
+      log("Failed to get extension versions");
+    }
+
+    // If the widget was removed from the tree while the asynchronous platform
+    // message was in flight, we want to discard the reply rather than calling
+    // setState to update our non-existent appearance.
+    if (!mounted) return;
+
+    setState(() {
+      _identityVersion = identityVersion;
+    });
+  }
+
+  Future<void> appendUrl() async {
+    String result = "";
+
+    try {
+      result = await Identity.appendToUrl("www.myUrl.com");
+    } on PlatformException {
+      log("Failed to append URL");
+    }
+
+    if (!mounted) return;
+    setState(() {
+      _appendToUrlResult = result;
+    });
+  }
+
+  Future<void> getExperienceCloudId() async {
+    String result = "";
+
+    try {
+      result = await Identity.experienceCloudId;
+    } on PlatformException {
+      log("Failed to get experienceCloudId");
+    }
+
+    if (!mounted) return;
+    setState(() {
+      _experienceCloudId = result;
+    });
+  }
+
+  Future<void> syncIdentifiers() async {
+    Identity.syncIdentifiers({
+      "idType1": "idValue1",
+      "idType2": "idValue2",
+      "idType3": "idValue3",
+    });
+  }
+
+  Future<void> syncIdentifiersWithAuthState() async {
+    Identity.syncIdentifiersWithAuthState(
+      {
+        "idType1": "idValue1",
+        "idType2": "idValue2",
+        "idType3": "idValue3",
+      },
+      MobileVisitorAuthenticationState.authenticated,
+    );
+  }
+
+  Future<void> syncIdentifier() async {
+    Identity.syncIdentifier(
+      "idType1",
+      "idValue1",
+      MobileVisitorAuthenticationState.authenticated,
+    );
+  }
+
+  Future<void> getUrlVariables() async {
+    String result = "";
+
+    try {
+      result = await Identity.urlVariables;
+    } on PlatformException {
+      log("Failed to get url variables");
+    }
+
+    if (!mounted) return;
+    setState(() {
+      _getUrlVariablesResult = result;
+    });
+  }
+
+  Future<void> getIdentifiers() async {
+    late List<Identifiable> result;
+
+    try {
+      result = await Identity.identifiers;
+    } on PlatformException {
+      log("Failed to get identifiers");
+    }
+
+    if (!mounted) return;
+    setState(() {
+      _getIdentifiersResult = result.toString();
+    });
+  }
+
+  // // UTIL
+  RichText getRichText(String label, String value) {
+    return new RichText(
+      text: new TextSpan(
+        // Note: Styles for TextSpans must be explicitly defined.
+        // Child text spans will inherit styles from parent
+        style: new TextStyle(
+          fontSize: 14.0,
+          color: Colors.black,
+        ),
+        children: <TextSpan>[
+          new TextSpan(
+              text: label, style: new TextStyle(fontWeight: FontWeight.bold)),
+          new TextSpan(text: value),
+        ],
+      ),
+    );
+  }
```

Comment:
we can now import util and remove this as well

Reply:
Thanks, I missed this one.

Reply:
I also added License to the sample app.